libs/api/domains/form-system/src/models/applications.model.ts

```
@@ -72,3 +72,18 @@ export class Application {
   @Field(() => [Section], { nullable: 'itemsAndList' })
   sections?: Section[]
 }
+
+@ObjectType('FormSystemApplicationResponse')
+export class ApplicationResponse {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'ApplicationResponse'. Consider to delete it or rename it.

'ApplicationResponse' is defined here:


(lint/suspicious/noRedeclare)